### PR TITLE
update changefile and add beta tags to new interfaces

### PR DESCRIPTION
**Pull request by Jessica-Koch**
update the changefile with more detail and add beta tags to new interfaces

### CODE REVIEW

**Comment by erinha** [comment]
Thanks for making this change! Can you also add a beta tag on the `actionInfo` property itself on the `app.Context` interface?

**Review by erinha — CHANGES_REQUESTED** [review]
Please add beta tag to `app.Context.actionInfo` property. Also (ironically?) you need a change file for updating the change file 🤣

File: change/@microsoft-teams-js-c5770fed-2b55-42a7-987b-1618dd76629b.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "update changefile and add beta tags to new interfaces",
```
Comment (erinha) [inline change/@microsoft-teams-js-c5770fed-2b55-42a7-987b-1618dd76629b.json:3]: Can you please actually change this to:

Added `@beta` tags to new content action-related interfaces.

I don't think we need to mention the change file part for our change log. :)

Comment (Jessica-Koch) [reply change/@microsoft-teams-js-c5770fed-2b55-42a7-987b-1618dd76629b.json]: Of course

**Review by erinha — APPROVED** [review]
🚀

**Review by TrevorJoelHarris — APPROVED** [review]

**Pull request merged**